compiler/cpp/src/generate/t_php_generator.cc

```diff
@@ -1720,6 +1720,7 @@ void t_php_generator::generate_service_client(t_service* tservice) {
                << "TBinaryProtocolAccelerated) && function_exists('thrift_protocol_write_binary');"
                << endl;
 
+    f_service_client << indent() << "$this->seqid_ = rand();" << endl;
     f_service_client << indent() << "if ($bin_accel)" << endl;
```

Comment:
I'm not quite sure what that ```rand()``` should do here? The sequential number is expected to be sequential. I would be a bit surprised when a function called ```rand()``` starts to return sequential numbers.